### DISCUSSION TITLE
Icon Request: icon-dancing-couple

### DISCUSSION THREAD

**Discussion by JuanObiJuan** [FortAwesome/Font-Awesome#21210] [Icon Requests]
Hi.

I think that it will be very useful (at least for me) to have an icon to express a party with people dancing.

Maybe it could be used to like a reunion with friends o simple like something like you enjoy with friends.

Maybe, its not very good idea to include people drinks as is not for everybody but I think that people dancing like here http://bambiemultimedia.com/bambiemultimedia/images/106px-party_icon.svg.png

or people with funny hats like botton left here

http://st.depositphotos.com/1029662/1230/v/950/depositphotos_12307466-Friend-Party-Celebration-Birthday-Icon-Symbol-Sign-Pictogram.jpg

I was looking for some icons but i could not find something with similar meanings.

Thank you.

**Comment by JacobCWard** [comment]
👍

**Comment by vellvoid** [comment]
Is this one coming to FA 6 perhaps? It appears pretty useful and works with other artistic icons such as music or theatre. When you search dance on fa's site you get.. dumpster fire. IMHO dance is much more needed than a dumpster file 🤷. Not to mention it's been open since 2013. Don't get me wrong, appreciate your work keep it up and stay safe ;). (but give us a dance icon)

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
